examples/ChatQnA/deploy/gaudi.md

```diff
@@ -250,6 +257,8 @@ Check if you have the below set of docker images, before moving on to the next s
 :::
 ::::
 
+:::

```

Comment:
Looks like you've got an extra set of closing fences.  Delete these two lines since the ones above are sufficient.

Reply:
removed